docs/3-limit-order/2-limit-order-v2-endpoints.md

```diff
@@ -0,0 +1,208 @@
+
+# Create limit order
+Sends a POST request to the Jupiter Limit Order API to get the unsigned transactions needed to create an order.
+
+<details>
+  <summary>
+    <div>
+      <div className="api-method-box post">POST</div>
+      <p className="api-method-path">https://api.jup.ag/limit/v2/createOrder</p>
+    </div>
+  </summary>
+
+**Parameters in use in the below code example:**
+
+- `inputMint`: The mint address of the input token (required).
+- `outputMint`: The mint address of the output token (required).
+- `maker`: 
+- `payer`: 
+- `makingAmount`: 
+- `takingAmount`
+- `expiredAt`
+- `feeBps`
+- `computeUnitPrice`
+- `referral`
+- `inputTokenProgram`
+- `outputTokenProgram`
+- `wrapAndUnwrapSol`
+
+### Example response
+```typescript
+type CancelOrders = {
+  maker: string;
+
+  // "auto" sets the priority fee based on network congestion
+  // and it will be capped at 500,000
+  computeUnitPrice: string | "auto";
+
+  // Specific order account public keys to cancel/close
+  orders?: string[] | undefined;
+};
+
+type CancelOrdersResponse = {
+  txs: string[];
+};
```

Comment:
Header should be "Create order request body and response" with the `CreateOrder` and `CreateOrderResponse` types.

Example response should be the following:

```suggestion
{
  "order": "EWxFUCUAzvo2siM9gYVPWkaouvCfemwnwkfdVw2ib7JH",
  "tx": "AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAQAJDgljyvLecNltZyLKJwcNk/0wjQipK4AUImvf2FZSxHIzDYJ0JdiFdLKUD0/uyu862mvKMrRGcmeB0hacZldr91rI1D6IwBADhxzCinLz0WuLrIJDCWIg/AajAnWXFcf8wi6Dlq1U1ClYCNsR8Xj0JlqSi9FGU0GP64HMzvBB8o/7Pc01wNmmSKitJONFP3gmdGbtde+pcSLhy7GOlMkI03UDBkZv5SEXMv/srbpyw5vnvIzlu8X3EmssQ5s6QAAAAIyXJY9OJInxuz0QKRSODYMLWhOZ2v8QhASOe9jb6fhZBpuIV/6rgYT7aH9jRhjANdrEOdwa6ztVmKDwAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbd9uHXZaGT2cvhRs7reawctIXtX1s3kTqM9YV+/wCpCsNKlsFmcVpgwSM+yiWKDfMLHshY4HRcc2oSYmZjSyIQiYnNzGGPBnZ0rGd+KZqtsSj+INnM2CjlCMnxv5sw68b6evO+2606PWXzaqvJdDGxu+TC0vbg5HymAgNFL11hkn9sh+xDEAWX2S7IDh4PwuUsi7/8dQ+DB2YEV+4zsuGS8LiUcikk5FUuAM3vpZjsAmrMZL3FPOcH2v4SeX3/fwQFAAUCoAMCAAUACQNd9AgAAAAAAAYGAAEABwgJAQEKDwAAAgMECwoMBwkJCAYNCiKFbkqvcJ/1n9vKKe2gGX+OAOH1BQAAAACA8PoCAAAAAAAAAA=="
}
```